lib/braintree/webhook_testing_gateway.rb

```diff
@@ -163,26 +163,25 @@ def _merchant_account_declined_sample_xml(id)
       <<-XML
           <api-error-response>
               <message>Credit score is too low</message>
-              <errors>
```

Comment:
Removing this tag results in the opening `<api-error-response>` tag to be unterminated.